server/graphql/v2/input/TierUpdateInput.ts

```diff
@@ -36,13 +39,13 @@ export const TierUpdateInput = new GraphQLInputObjectType({
       type: AmountInput,
     },
     type: {
-      type: new GraphQLNonNull(TierType),
+      type: TierType,
     },
     amountType: {
-      type: new GraphQLNonNull(TierAmountType),
+      type: TierAmountType,
     },
     frequency: {
-      type: new GraphQLNonNull(TierFrequency),
+      type: TierFrequency,
```

Comment:
Before this PR, partial updates were supported with some fields but not others. This change makes them officially supported, it should be valid to call a mutation like:

```es6
editTier(tier: { id: 'xxx', name: 'New name')
```

to update just one attribute.